todo_app/frontend/apps.py

```diff
@@ -0,0 +1,5 @@
+from django.apps import AppConfig
+
+
```

Comment:
All these frontend files should be removed. Your frontend is now a separate entity.